go.mod

```diff
@@ -122,7 +122,7 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0 // indirect
 	github.com/mitchellh/mapstructure v1.4.3 // indirect
 	github.com/moby/locker v1.0.1 // indirect
-	github.com/moby/sys/mountinfo v0.6.1 // indirect
+	github.com/moby/sys/mountinfo v0.6.2-0.20220518012833-28ef7ed18e6a // indirect
```

Comment:
Requesting the v0.6.2 release: https://github.com/moby/sys/pull/114#issuecomment-1129849702